intranet/static/js/common.nav.js

```diff
@@ -0,0 +1,39 @@
+$(function() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["$" is not defined.](https://www.codacy.com/app/tjcsl/ion/file/2286322831/issues/source?bid=3176160&fileBranchId=3189731#l1)